packages/react/src/components/OverflowSet/OverflowSet.test.tsx

```diff
@@ -4,14 +4,15 @@ import * as React from 'react';
 import { create } from '@fluentui/utilities/lib/test';
 import * as ReactTestUtils from 'react-dom/test-utils';
 import { CommandBarButton } from '../../Button';
-import { IKeytipProps } from '../../Keytip';
 import { KeytipLayer, KeytipLayerBase } from '../../KeytipLayer';
 import { arraysEqual, find } from '../../Utilities';
-import { IUniqueKeytip, KeytipManager, ktpTargetFromId } from '../../Keytips';
+import { KeytipManager, ktpTargetFromId } from '../../Keytips';
 import { OverflowSet } from './OverflowSet';
-import { IOverflowSetItemProps } from './OverflowSet.types';
 import * as path from 'path';
 import { isConformant } from '../../common/isConformant';
+import type { IKeytipProps } from '../../Keytip';
+import type { IUniqueKeytip } from '../../Keytips';
```

Comment:
not this PR's fault but...why? 😭

Reply:
JD touched it last 2 years ago, clearly his fault!